phys2denoise/metrics/cardiac.py

```diff
@@ -26,59 +26,93 @@ def heart_beat_interval(card, peaks, samplerate, window=6, central_measure="mean
     window : float, optional
         Size of the sliding window, in seconds.
         Default is 6.
-    central_measure : "mean", "median", string, optional
+    central_measure : "mean","average", "avg", "median", "mdn",  string, optional
         Measure of the center used (mean or median).
         Default is "mean".
+    cardiac_metrics : "hbi", "hrv", "hbi_hrv",  string
+        Cardiac metric(s) to calculate.
     Returns
     -------
-    hbi : 2D numpy.ndarray
+    hbi_out : 2D numpy.ndarray
+        Heart Beats Interval values.
+        The first column is raw HBI values, in seconds.
+        The second column is HBI values convolved with the CRF, cut to the length
+        of the raw HBI values, in seconds.
+
+    hbi_out_padd : 2D numpy.ndarray
+        Heart Beats Interval values.
+        The first column is raw HBI values, in seconds, padded to the length of the
+        initial convolved HBI values, in seconds.
+        The second column is HBI values convolved with the CRF, in seconds
+
+    hrv_out : 2D numpy.ndarray
+        Heart Rate Variability values.
+        The first column is raw HRV values, in seconds.
+        The second column is HRV values convolved with the CRF, cut to the length
+        of the raw HBI values in seconds.
+
+    hrv_out_padd : 2D numpy.ndarray
         Heart Beats Interval values.
-        The first column is raw HBI values.
-        The second column is HBI values convolved with the RRF.
+        The first column is raw HRV values, in seconds, padded to the length of the
+        initial convolved HRV values, in seconds.
+        The second column is HRV values convolved with the CRF, in seconds
 
     Notes
     -----
     Heart beats interval (HBI) was introduced in [1]_, and consists of the
     average of the time interval between two heart beats based on ppg data within
-    a 6-second window.
-
+    a 6-seconds window.
     This metric is often lagged back and/or forward in time and convolved with
     an inverse of the cardiac response function before being included in a GLM.
 
+    Heart rate variability (HRV) was introduced in [1]_, and consists of the average
+    variation of the time interval between to heart beats based on ppg data within
+    a 6-seconds window.
+    IMPORTANT : Here both cardiac metrics have a meaning since they are computed in seconds.
```

Comment:
```suggestion
    IMPORTANT : Here both metrics' unit of measure have meaning, since they are based on seconds. Hence, zscoring might remove important quantifiable information
```